Source/DataAccessLayer/Bases/EntityProviderBaseCoreClass.generated.cst

```diff
@@ -403,925 +403,949 @@ namespace <%= DALNameSpace %>.Bases
+        /// <param name="start">Row number at which to start reading.</param>
+        /// <param name="pageLength">Number of rows to return.</param>
+        /// <param name="count">out. The number of rows that match this query.</param>
+        /// <returns>Returns a typed collection of Entity objects.</returns>
+        public virtual <%=listName%><Entity> <%= MethodNames.Find %>(IFilterParameterCollection parameters, string orderBy, int start, int pageLength, out int count)
+        {
+            return <%= MethodNames.Find %>((TransactionManager) null, parameters, orderBy, start, pageLength, out count);
+        }
+        
+        /// <summary>
+        /// Returns rows from the DataSource that meet the parameter and sortColumn conditions.
+        /// </summary>
+        /// <param name="parameters">A collection of <see cref="SqlFilterParameter"/> or <see cref="ParameterizedSqlFilterBuilder&lt;T&gt;"/> objects.</param>
+        /// <param name="sortColumns">A collection of <see cref="SqlSortColumn"/> or <see cref="SqlSortBuilder&lt;T&gt;"/> objects</param>
+        /// <param name="start">Row number at which to start reading.</param>
+        /// <param name="pageLength">Number of rows to return.</param>
+        /// <param name="count">out. The number of rows that match this query.</param>
+        /// <returns>Returns a typed collection of Entity objects.</returns>
+        public virtual <%=listName%><Entity> <%= MethodNames.Find %>(IFilterParameterCollection parameters, ISortColumnCollection sortColumns, int start, int pageLength, out int count)
+        {
+            return <%= MethodNames.Find %>((TransactionManager) null, parameters, sortColumns.ToString(), start, pageLength, out count);
+        }
+        
+        /// <summary>
+        /// Returns rows from the DataSource that meet the parameter conditions.
+        /// </summary>
+        /// <param name="transactionManager"><see cref="TransactionManager"/> object</param>
+        /// <param name="parameters">A collection of <see cref="SqlFilterParameter"/> or <see cref="ParameterizedSqlFilterBuilder&lt;T&gt;"/> objects.</param>
+        /// <param name="orderBy">Specifies the sort criteria for the rows in the DataSource (Name ASC; BirthDay DESC, Name ASC);</param>
+        /// <param name="start">Row number at which to start reading.</param>
+        /// <param name="pageLength">Number of rows to return.</param>
+        /// <param name="count">out. The number of rows that match this query.</param>
+        /// <returns>Returns a typed collection of Entity objects.</returns>
+        public virtual <%=listName%><Entity> <%= MethodNames.Find %>(TransactionManager transactionManager, IFilterParameterCollection parameters, string orderBy, int start, int pageLength, out int count)
+        {
+            <% // TODO: make this method abstract after it has been fully implemented! %>

```

Comment:
Was this meant to be implemented?

Reply:
Yes, this is the supporting class for deep loads.  My changes here, assign from converting tabs to spaces, are in support of the new deep load features, adding the Flag attribute to the Enum, and granting visibility of the load depth to the event triggered.